### PR TITLE
Typescript -> TypeScript (spanish)

### PR DESCRIPTION
**Pull request by lipis**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Related to issue #35185

### CODE REVIEW

File: guide/spanish/typescript/getters-setters/index.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
-localeTitle: Hechiceros y Setters
----
# Hechiceros y Setters
+---
+title: Getters & Setters
+localeTitle: Hechiceros y Setters
+---
 
-Typescript también admite `get` y `set` propiedades. Las propiedades Get y Set en realidad se llaman Accessors. Los que acceden a una propiedad contienen declaraciones ejecutables asociadas con obtener (leer) o configurar (escribir) la propiedad. Las declaraciones pueden contener get accessor o set accessor o ambos.
+# Hechiceros y Setters
```
Comment (hestellez) [inline guide/spanish/typescript/getters-setters/index.md]: You could change the Hechiceros to Getters

**Review by hestellez — CHANGES_REQUESTED** [review]

File: guide/spanish/angular/index.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ localeTitle: Angular
 
 AngularJS (versiones 1.x) es un Framework de código abierto basado en JavaScript. Es multiplataforma y se utiliza para desarrollar una aplicación web de una sola página (SPWA). AngularJS implementa el patrón MVC para separar los componentes de lógica, presentación y datos. También utiliza la inyección de dependencia para hacer uso de los servicios del lado del servidor en las aplicaciones del lado del cliente.
 
-Angular (versiones 2.x y posteriores) es un marco de código abierto basado en Typescript para desarrollar aplicaciones web front-end. Angular tiene las siguientes funciones como genéricos, escritura estática y también algunas características de ES6.
+Angular (versiones 2.x y posteriores) es un marco de código abierto basado en TypeScript para desarrollar aplicaciones web front-end. Angular tiene las siguientes funciones como genéricos, escritura estática y también algunas características de ES6.
```
Comment (brianRey-glb) [inline guide/spanish/angular/index.md]: You can replace it with 

> Angular (versiones 2.x y posteriores) es un framework de código abierto basado en TypeScript utilizado el desarrollo de aplicaciones web front end. Angular posee funcionalidades tales como genéricos, tipado estático y muchas de las características de ES6.

**Comment by RandellDawson** [comment]
@lipis Can you please make the changes suggested by the users and fix the merge conflict?

Thank you.

**Comment by RandellDawson** [comment]
After reviewing the PR again, since the scope of the PR was just to fix the "TypeScript" instances, just ignore the other change requests.  This PR LGTM.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**